test/index.js

```diff
@@ -27,17 +27,34 @@ if (typeof global.gc !== 'function') {
   process.exit(process.exitCode);
 }
 
+const testModules = [];
+
 const fs = require('fs');
 const path = require('path');
 
-let testModules = [];
+let filterCondition = process.env.npm_config_filter || '';
+let filterConditionFiles = [];
+
+if (process.env.npm_config_filter !== null && process.env.npm_config_filter !== undefined && process.env.npm_config_filter.trim() !== '') {
+  filterCondition = require('../unit-test/matchModules').matchWildCards(process.env.npm_config_filter);
+  filterConditionFiles = filterCondition.split(' ').length ? filterCondition.split(' ') : [filterCondition];
```

Comment:
You should make this check explicitly `> 0`.

Reply:
agreed